src/HealthChecks.AzureServiceBus/Configuration/AzureEventHubOptions.cs

```diff
@@ -0,0 +1,41 @@
+using Azure.Core;
+using Azure.Messaging.EventHubs;
+
+namespace HealthChecks.AzureServiceBus.Configuration;
+
+/// <summary>
+/// Azure Event Hub configuration options.
+/// </summary>
+public class AzureEventHubOptions
+{
+    /// <summary>
+    /// The azure event hub connection string.
+    /// </summary>
+    public string? ConnectionString { get; set; }
+
+    /// <summary>
+    /// The azure event hub name.
+    /// </summary>
+    public string? EventHubName { get; set; }
+
+    /// <summary>
+    /// The azure event hub fully qualified namespace.
+    /// </summary>
+    /// <remarks>
+    /// Must be used in conjunction with the <see cref="Credential"/> property.
+    /// </remarks>
+    public string? Endpoint { get; set; }
```

Comment:
`EventHubProducerClient` takes `fullyQualifiedNamespace` as the first aergument but here it's named as `Endpoint`. Maybe to change name to `FullyQualifiedNamespace`?

Reply:
I've seen the term Endpoint being used in different packages for this, but I agree, FullyQualifiedNamespace is more correct for Service Bus and Event Hub. I can change that.